prow/tide/tide.go

```diff
@@ -179,18 +180,90 @@ func byRepoAndNumber(prs []PullRequest) map[string]PullRequest {
 	return m
 }
 
+func requirementDiff(pr *PullRequest, q *config.TideQuery) (string, int) {
```

Comment:
I suggest a quick comment describing this function, because I can't tell quite what it does based on the name and types.

Reply:
Done.